app/assets/javascripts/collections.js

```diff
@@ -0,0 +1,27 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+$(document).ready(function(){
+  var institutionId = 0;
+
+  $('#institution_search').on('click', '.select-option', function(){
```

Comment:
Most of our JS has a space between `function()` and `{` (not sure if it's part of our style guide though)